Description: allow i386 builds on amd64 and set march to match gcc
Author: Steve Capper <steven.capper@gmail.com>

--- a/build/linux.gcc.inc
+++ b/build/linux.gcc.inc
@@ -91,7 +91,7 @@
 
 ifeq (ia32,$(arch))
     ITT_NOTIFY = -DDO_ITT_NOTIFY
-    CPLUS_FLAGS += -m32 -march=pentium4 $(RTM_KEY)
+    CPLUS_FLAGS += -m32 $(RTM_KEY)
     LIB_LINK_FLAGS += -m32
 endif
 
--- a/build/linux.inc
+++ b/build/linux.inc
@@ -63,6 +63,9 @@
         ifeq ($(deb_host_arch),x32)
                 export arch:=x32
         endif
+        ifeq ($(deb_host_arch),i386)
+                export arch:=ia32
+        endif
         ifndef arch
                 export arch:=$(uname_m)
                 $(warning "Unknown arch: " $(arch))
